### ISSUE TITLE
EfficientNet-b0 model card

### ISSUE THREAD

**Issue by keerthana-r-mcw** [tenstorrent/tt-metal#13047]
**Brief executive summary :**
- Reference- https://github.com/lukemelas/EfficientNet-PyTorch
- Graph - [efficientnet-b0.pdf](https://github.com/user-attachments/files/17113603/efficientnet-b0.pdf)
- Unit_test : [sabira/unit_test_efficientnetb0](https://github.com/tenstorrent/tt-metal/compare/main...sabira/unit_test_efficientnetb0)
- Branch: [sabira/efficientnetb0_ttnn](https://github.com/tenstorrent/tt-metal/tree/sabira/efficientnetb0_ttnn)

**Modules**:
MBConvBlock
ConvDynamicSamePadding

**Ops**:
1. Conv
2. BatchNorm
3. Sigmoid
4. Adaptive average pool
5. Linear

**FPS**
Single device: 74 FPS 
Multi device: 146 FPS

**Issues**
- EfficientNetB0 Trace having Low FPS - [#26225](https://github.com/tenstorrent/tt-metal/issues/26225)
